packages/vscode-extension/lib/runtime.js

```diff
@@ -2,8 +2,16 @@
 // the things ot requires may interfere with other modules that depend on the loading
 // order. In order to avoid issues related to that, we only require minimal set of
 // dependencies, and we load the main bits of runtime lazyli as a part of wrapper.js
-const AppRegistry = require("react-native/Libraries/ReactNative/AppRegistry");
-const parseErrorStack = require("react-native/Libraries/Core/Devtools/parseErrorStack");
+let AppRegistry = require("react-native/Libraries/ReactNative/AppRegistry");
+let parseErrorStack = require("react-native/Libraries/Core/Devtools/parseErrorStack");
+
+// since version 0.79 react native changed how imports of some modules work 
+// (https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0790-rc0)
+// we handle it mostly by spiting react native internal imports in babel_transformer.js
+// but this method works only for lazy imports and we want to require those modules 
+// here directly so we check if the module is a default export in runtime here. 
+AppRegistry = AppRegistry.default !== undefined ? AppRegistry.default : AppRegistry;
+parseErrorStack = parseErrorStack.default !== undefined ? parseErrorStack.default : parseErrorStack; 
```

Comment:
Can we export this from `rn-internals` to have one unified way of importing stuff from RN (that changes between versions)?
Otherwise, maybe we'd wrap this in some helper (`requireMaybeDefault(...)`) so it's a bit more clear what's going on, without having to read a 5 line comment.